bot/cogs/games/games.py

```diff
@@ -39,3 +40,32 @@ async def ball8(self, ctx: Context, *, question: str) -> None:
         embed.add_field(name="Answer", value=answer)
 
         await ctx.send(embed=embed)
+
+    @command(aliases=["pokesearch"])
+    async def pokemon(self, ctx: Context, pokemon: str) -> None:
+        """
+        Fetches data about a given pokemon eg. pokemon pikachu.
+        """
+
+        async with aiohttp.ClientSession() as session:
+            async with session.get(f"https://pokeapi.co/api/v2/pokemon/{pokemon}") as resp:
+                data = await resp.json()
+
+        pokemon_embed = Embed(
+            title=f"{pokemon.capitalize()} Info",
+            color=Color.blurple()
+        )
+
+        ability_names = [f"`{ability['ability']['name']}`" for ability in data["abilities"]]
+        pokemon_types = [f"`{ptype_raw['type']['name']}`" for ptype_raw in data["types"]]
+        base_stat_names = ["Hp", "Attack", "Defence", "Special-Attack", "Special-Defence", "Speed"]
+        base_stats_zip = zip(base_stat_names, data["stats"])
+        base_stats = [f"**{stat_name}**: `{str(base_stat_dict['base_stat'])}`" for stat_name, base_stat_dict in base_stats_zip]
+
+        pokemon_embed.set_thumbnail(url=data["sprites"]["front_default"])
+        pokemon_embed.add_field(name="Base Stats", value="❯❯ " + "\n❯❯ ".join(base_stats))
+        pokemon_embed.add_field(name="Type", value="❯❯ " + "\n❯❯ ".join(pokemon_types))
+        pokemon_embed.add_field(name="Weight", value=f"❯❯ `{str(data['weight'])}`")
+        pokemon_embed.add_field(name="Abilities", value="❯❯ " + "\n❯❯ ".join(ability_names), inline=True)
```

Comment:
```suggestion
        pokemon_embed.add_field(name="❯❯Base Stats", value="\n ".join(base_stats))
        pokemon_embed.add_field(name="❯❯Type", value="\n".join(pokemon_types))
        pokemon_embed.add_field(name="❯❯Weight", value=f"`{str(data['weight'])}`")
        pokemon_embed.add_field(name="❯❯Abilities", value="\n".join(ability_names), inline=True)
```
I meant you to put `❯❯` in the field names, not body